package.json

```diff
@@ -45,7 +45,7 @@
   },
   "dependencies": {
     "@stoplight/json": "^2.2.2",
-    "@stoplight/types": "^9.1.2",
+    "@stoplight/types": "git://github.com/stoplightio/types.git#fix/do-not-flatten-security",
```

Comment:
Let's use npm version of types.

Reply:
@P0lip, we don't have npm version of types with the changes in it (`IHttpOperation`), or do we?

Reply:
Let's release it first and then bump the version.